tutorials/cp-starter-isuite-onboard-subscribe/rules.vr

```diff
@@ -0,0 +1,14 @@
+[VALIDATE_4]
```

Comment:
Please remove this file from your pull request. You should only push this file to Tutorials-Contribution.